test-bin/echo/echo_bench.ml

```diff
@@ -0,0 +1,46 @@
+
+open Lwt.Infix
+
+open Capnp_rpc_lwt 
+
+let () =
+  Logs.set_level (Some Logs.Info);
+  Logs.set_reporter (Logs_fmt.reporter ())
+
+let rec create_for v = function
```

Comment:
I think this is just [`List.init`](https://sanette.github.io/ocaml-api/4.10/List.html#VALinit) with the arguments swapped.

Reply:
Did not know that existed either... I had thought it was an odd lack.